lib/node_modules/@stdlib/stats/base/dvarmpn/src/main.c

```diff
@@ -31,12 +32,34 @@
 * @param mean        mean
 * @param correction  degrees of freedom adjustment
 * @param X           input array
-* @param stride      stride length
+* @param strideX      strideX length
```

Comment:
```suggestion
* @param strideX      stride length
```
kindly also adjust the spacing of the descriptions here. ideally, we would want them to be stacked over one another.

Reply:
Resolved